application/org.openjdk.jmc.console.agent/plugin.xml

```diff
@@ -96,4 +96,43 @@
       </command>
     </menuContribution>
   </extension>
+     <extension
```

Comment:
There should be a comma after the year(s) in the copyright statement in this file. I know that this hasn't changed, but let's take the opportunity to fix this. :)